src/chi/components/data-table/data-table.scss

```diff
@@ -32,6 +32,12 @@ $bg-url: $bg-url + ' fill="%230262b9" fill-rule="evenodd"/%3E%3C/svg%3E';
   overflow-y: initial;
   width: 100%;
 
+  .-d-print--only {
+    .chi-data-table__print-footer {
+      padding: 1rem;
+    }
+  }
+
```

Comment:
```suggestion
  .chi-data-table__print-footer {
    padding: 1rem;
  }
```
As we agreed, `-d-print--only` is a generic utility class. We never attach CSS properties to classes associated with any kind of utility classes.